tests/neptune/management/internal/test_utils.py

```diff
@@ -42,3 +47,35 @@ def test_normalize_project_name(self):
 
         with self.assertRaises(ConflictingWorkspaceName):
             normalize_project_name(name="jackie/sandbox", workspace="john")
+
+
+@pytest.mark.parametrize(
```

Comment:
According to docs, the key should be uppercase, so lowercase input is perfect test data, but I'd expect upercase result.

Reply:
Good point, fixed